node/service/chain-specs/example.json

```diff
@@ -108,6 +138,14 @@
       "authorityDiscovery": {
         "keys": []
       },
+      "society": {
```

Comment:
The pallet-society was removed in [PR#275](https://github.com/Cerebellum-Network/blockchain-node/pull/275)